docs/source/techniques/ISISPowder-OSIRIS-v1.rst

```diff
@@ -0,0 +1,307 @@
+If not more details can be read here: :ref:`intro_to_objects-isis-powder-diffraction-ref`
+
+To create a OSIRIS object the following parameters are required:
+
+- :ref:`calibration_directory_osiris_isis-powder-diffraction-ref`
+- :ref:`output_directory_osiris_isis-powder-diffraction-ref`
+- :ref:`user_name_osiris_isis-powder-diffraction-ref`
+
+Optionally a configuration file may be specified if one exists
+using the following parameter:
+
+- :ref:`config_file_osiris_isis-powder-diffraction-ref`
+
+See :ref:`configuration_files_isis-powder-diffraction-ref`
+on YAML configuration files for more details
+
+Example
+^^^^^^^
+
+..  code-block:: python
+
+  from isis_powder.osiris import Osiris
+
+  calibration_dir = r"C:\path\to\calibration_dir"
+  output_dir = r"C:\path\to\output_dir"
+  config_dir = r"C:\path\to\config.yaml"
+
+  osiris_example = Osiris(user_name="Calib",
+                          calibration_directory=calibration_dir,
+                          output_directory=output_dir,
+                          config_file=config_dir)
+
+Methods
+--------
+The following methods can be executed on a OSIRIS object:
+
+- :ref:`run_diffraction_focusing_osiris_isis-powder-diffraction-ref`
+
+For information on creating a OSIRIS object see:
+:ref:`creating_osiris_object_isis-powder-diffraction-ref`
+
+.. _run_diffraction_focusing_osiris_isis-powder-diffraction-ref:
+
+run_diffraction_focusing
+^^^^^^^^^^^^^^^^^^^^
+The *diffraction_focusing* method allows a user to process a series of runs into a
+focused dSpacing workspace. Whilst processing the runs the scripts can apply any corrections
+the user enables.
+
+On OSIRIS the following parameters are required when executing *create_vanadium*:
+
+- :ref:`calibration_mapping_file_osiris_isis-powder-diffraction-ref`
+- :ref:`vanadium_normalisation_osiris_isis-powder-diffraction-ref`
+- :ref:`subtract_empty_instrument_osiris_isis-powder-diffraction-ref`
+- :ref:`merge_drange_osiris_isis-powder-diffraction-ref`
+
+Example
+=======
+..  code-block:: python
+
+  # Notice how the filename ends with .yaml
+  cal_mapping_file = r"C:\path\to\cal_mapping.yaml"
+
+  osiris_example.run_diffraction_focusing(run_number="119977-119988",
+                                          merge_drange=True,
+                                          subtract_empty_instrument=True,
+                                          vanadium_normalisation=True,
+                                          calibration_mapping_file=cal_mapping_file)
+.. _calibration_mapping_osiris-isis-powder-ref:
+
+Calibration Mapping File
+-------------------------
+The calibration mapping file holds the mapping between
+run numbers, current label, offset filename and the empty
+and vanadium numbers.
+
+For more details on the calibration mapping file see:
+:ref:`cycle_mapping_files_isis-powder-diffraction-ref`
+
+The layout on OSIRIS should look as follows for each block
+substituting the below values for appropriate values:
```

Comment:
is this sentence needed? The previous one says that the layout is an example, so implies that the values will change